### DISCUSSION TITLE
ping from device to loacal network

### DISCUSSION THREAD

**Discussion by aliakseika** [openthread/openthread#9077] [Q&A] [answered]
hello, maybe someone came across. Installed otbr, connected one device to the network. The essence of the thread has an ip of the form fdb0:a7b3:ac73:a4eb: In the local network, I also got an IPv6 network fd11:1111:1122:222 Is there any way to get through from the fdb0 network to fd11? Or is it basically impossible?

**Comment by jwhui** [comment, marked as answer]
As long as the Thread network and adjacent infrastructure link have global-scope IPv6 prefixes configured, the Thread Border Router can route traffic between them. Check out this codelab: [Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery](https://openthread.io/codelabs/openthread-border-router).

**Reply by aliakseika** [reply]
thanks, it's working now.

**Reply by jwhui** [reply]
Great to hear. Thanks for reporting back!
